test/integration/api/datasets.js

```diff
@@ -109,6 +112,7 @@ describe('datasets and entities', () => {
           .set('X-Extended-Metadata', 'true')
           .expect(200)
           .then(({ body }) => {
+            body[0].should.be.an.ExtendedDataset();
             body.map(({ createdAt, lastEntity, ...d }) => {
               createdAt.should.not.be.null();
               should(lastEntity).be.null();
```

Comment:
I don't think we need the `createdAt.should.not.be.null();` line but the last entity being null line can stay.